### PR TITLE
change inject error's directories

### PR DESCRIPTION
**Pull request by Pslydhh**
## What type of PR is this：
- [ ] bugfix
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Problem Summary(Required) ：

Change default directories of inject errors.

### CODE REVIEW

**Review by Seaven — DISMISSED** [review]

File: be/src/common/config.h [hunk 1]
```diff
@@ -810,12 +810,14 @@ CONF_Int32(internal_service_async_thread_num, "10");
...
  */
-CONF_Int32(cardinality_of_inject, "100");
+CONF_Int32(cardinality_of_inject, "10");
 
 /*
  * Config range for inject erros,
  * Specify the source code directory,
  * Split by "," strictly.
  */
-CONF_String(directory_of_inject, "/src/exec,/src/exprs");
```
Comment (kangkaisen) [inline be/src/common/config.h:820]: why remove exprs Directory？

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: be/src/common/config.h (hunk 1 above)
Comment (Pslydhh) [reply be/src/common/config.h:820]: > why remove exprs Directory？

It may Interrupte sqlancer's execute.

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by kangkaisen — APPROVED** [review]

**Review by ZiheLiu — APPROVED** [review]

**Pull request merged**